### PR TITLE
docs: add documentation for item favorite, small refactor

### PR DESCRIPTION
**Pull request by pyphilia**
ref #1528

### CODE REVIEW

File: src/services/item/plugins/itemFavorite/schemas.ts [hunk 1]
```diff
@@ -35,38 +36,50 @@ export const packedFavoriteSchemaRef = registerSchemaAsRef(
...
-      },
-      {
-        additionalProperties: false,
-      },
-    ),
-  ),
+export const getOwnFavorite = {
+  operationId: 'getOwnFavorite',
+  tags: ['favorite'],
+  summary: 'Get all own bookmarked instances',
```
Comment (ReidyT) [inline src/services/item/plugins/itemFavorite/schemas.ts:48]: What about 
```suggestion
  summary: 'Get all bookmarked instances of the current member',
```

File: src/services/item/plugins/itemFavorite/schemas.ts [hunk 2]
```diff
@@ -35,38 +36,50 @@ export const packedFavoriteSchemaRef = registerSchemaAsRef(
...
+  description: 'Get all own bookmarked instances',
```
Comment (ReidyT) [inline src/services/item/plugins/itemFavorite/schemas.ts:49]:
```suggestion
  description: 'Get all bookmarked instances of the current member',
```

**Review by ReidyT — APPROVED** [review]
LGTM! I just have a global question: is the description mandatory? Because I have the feelings that in some case, it only add redundancy. Maybe we should only add the description when we add a real new information?

Here is from the example of Swagger 3, we see a real interest of the description, and I think they don't use it for each route.
image

**Comment by pyphilia** [comment]
Thanks for the review!
Openapi specs recommends to always have a summary and description for all endpoints!

**Comment by ReidyT** [comment]
> Thanks for the review! Openapi specs recommends to always have a summary and description for all endpoints!

Okay, thanks for your answer!

**Pull request merged**